esp-hal-embassy/build.rs

```diff
@@ -39,16 +39,70 @@ fn main() -> Result<(), Box<dyn Error>> {
     config.define_symbols();
 
     // emit config
-    generate_config(
+    let crate_config = generate_config(
         "esp_hal_embassy",
         &[(
             "low-power-wait",
             "Enables the lower-power wait if no tasks are ready to run on the thread-mode executor. This allows the MCU to use less power if the workload allows. Recommended for battery-powered systems. May impact analog performance.",
             Value::Bool(true),
             None
-        )],
+        ),
+        (
+            "timer-queue",
```

Comment:
Are we sure this shouldn't be a feature? Or because only one can be selected this is a use case for a config option?

Reply:
I thought we have a config system to use it, unless we can't for some reason. We allow non-power-of-two generic queue lengths, and that would be very inconvenient as a bunch of cargo features, and that kinda-sorta means to me that the other, related options should be handled similarly as esp-config options.

Reply:
I was making the comment to ensure we're not just emulating features in configs. Configs are more powerful than features and should be used _when_ they're appropriate. I think this is an appropriate case.